packages/compass-databases-navigation/package.json

```diff
@@ -34,7 +33,7 @@
   "scripts": {
     "bootstrap": "npm run compile",
     "prepublishOnly": "npm run compile && compass-scripts check-exports-exist",
-    "compile": "tsc -p tsconfig.json && gen-esm-wrapper . ./dist/.esm-wrapper.mjs",
+    "compile": "tsc -p tsconfig.json",
```

Comment:
No idea why it didn't compile anymore with the esm wrapper

Reply:
because you removed `"import": "./dist/.esm-wrapper.mjs",` above?